ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__multi-nightly.yaml

```diff
@@ -1313,6 +1313,52 @@ tests:
     test:
     - chain: openshift-e2e-test-qe
     workflow: cucushift-installer-rehearse-azure-ipi-publish-mixed-ingress-internal
+- as: baremetal-agent-ipv4-static-amd-mixarch-f14
+  cluster: build05
+  cron: 21 16 8,24 * *
+  steps:
+    cluster_profile: equinix-ocp-metal-qe
+    dependencies:
+      OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE: release:multi-latest
+    env:
+      ADDITIONAL_WORKER_ARCHITECTURE: aarch64
+      ADDITIONAL_WORKERS: "1"
+      ADDITIONAL_WORKERS_DAY2: "true"
+      AUX_HOST: openshift-qe-metal-ci.arm.eng.rdu2.redhat.com
+      DISCONNECTED: "false"
+      RESERVE_BOOTSTRAP: "false"
+      architecture: amd64
+      ipv4_enabled: "true"
+      ipv6_enabled: "false"
+      masters: "3"
+      workers: "0"
+    test:
+    - chain: openshift-e2e-test-qe
+    workflow: baremetal-lab-agent-install-network-static-day2
+- as: baremetal-agent-ipv4-static-disconnected-amd-mixarch-f14
```

Comment:
can we use dhcp instead of static and ADDITIONAL_WORKER_ARCHITECTURE_DAY2: false?

Reply:
Ok with dhcp instead of static.
What do you mean about "ADDITIONAL_WORKER_ARCHITECTURE_DAY2: false"?

Reply:
Can the agent installer generate two ISOs?

Reply:
@aleskandro me and @zniu1011 has discussed this question, we can't use agent ISO for additional nodes scale up, https://github.com/openshift/release/pull/47637#discussion_r1452891091

Reply:
@lwan-wanglin Fully correct.
@aleskandro Agent install only support one boot iso install, with fixed nodes' count, and fixed configurations. Any installation hasn't meet this fixed requirements, it will not pass checking and not go to install. Agent team was working on the day2 install design too, but it is impossible to follow the same agent way for day2. https://docs.google.com/document/d/1yXoU6QwS-zscxAvSbvrgP8KFe-MVZaScYYxc9ByVJ3A/edit#heading=h.zdxxupi1y8u3

Reply:
What I was wondering is: can we set all the nodes (even the additional ones) in the same agent-install config and generate one iso for each arch with the same exact content?

If we want to install an arm-based CP, how does the agent installer understand that it should generate an iso for arm? From the controlPlane.architecthre field?

Reply:
Yes, that is true, we are using controlPlane.architecthre and compute:- architecture field in the install-config to define the arch. 
For now, controlPlane.architecthre and compute:- architecture should be totally the same, if there are 2 different archs in install-config, the agent.<arch>.iso image can't created with error. 
We can run "create image" command twice to generate 2 sets: agent.x86_64.iso and agent.arrch64.iso, but it is not booting success to install the mixed arch clusters.

Reply:
Thanks for clarifying. Can you fix the documentation of the day2 parameter for the agent steps by specifying that only "true" is supported because day0 is not implemented yet?

Reply:
Sure, I will update the documentation to clarify this.